src/custom/state/affiliate/reducer.ts

```diff
@@ -25,6 +26,7 @@ export default createReducer(initialState, (builder) =>
       state.appDataHash = action.payload
     })
     .addCase(dismissNotification, (state, action) => {
-      state.isNotificationClosed = action.payload
+      state.isNotificationClosed = state.isNotificationClosed ?? {}
```

Comment:
In this case, I think `||` would be safer than `??`, given that if `state.isNotificationClosed` is ever `false` the next line will break, no?

Anyway, the type defines it either as an object or undefined, so it should be fine.